plugins/sparse_matrix_dataset.cc

```diff
@@ -159,8 +159,10 @@ struct SparseMatrixDataset::Itl
         }
 
         virtual const RowName & rowName(RowName & storage) const {
-            uint64_t i = internalStream->current();
-            return source->getRowNameTrans(RowHash(i), *trans);
+            throw HttpReturnException(500, "FIXME");
```

Comment:
`error: returning reference to local temporary object [-Werror,-Wreturn-stack-address]error: returning reference to local temporary object [-Werror,-Wreturn-stack-address]`